crates/re_viewer/src/ui/welcome_screen/welcome_page.rs

```diff
@@ -95,6 +99,7 @@ fn onboarding_content_ui(
                             include_str!("../../../data/quick_start_guides/rust_native.md"),
                             include_str!("../../../data/quick_start_guides/how_does_it_work.md"),
                         ],
+                        include_str!("../../../../../docs/code-examples/quick_start_connect.rs"),
```

Comment:
Oh yeah, this will mess up the publishing of `re_viewer` now, since it is referencing code outside the repository 🤦 

The quick-fix for this is to use symlinks 😬   - i.e. symlink `re_viewer/data/quick_start_connect.rs` to `docs/code-examples/quick_start_connect.rs`. It's not pretty, but it should work (we already use that trick for `re_web_viewer_server`).

Alternatively add `re_viewer/data/examples` as a second folder for `docs/code-examples/roundtrips.py` to check